concrete/src/Form/Service/Widget/DateTime.php

```diff
@@ -302,12 +302,15 @@ public function date($field, $value = null, $calendarAutoStart = true)
         if ($requestValue !== null) {
             $dateTime = $requestValue;
         } elseif ($value) {
-            if (!$value instanceof PHPDateTime) {
-                try {
-                    $dateTime = $dh->toDateTime($value);
-                } catch (Exception $x) {
-                }
-            }
+            if ($value instanceof PHPDateTime) {
+              $dateTime = clone $value;
+              $dateTime->setTimezone($dh->getTimezone('user'));
+            } else {
+              try {
+                $dateTime = $dh->toDateTime($value, 'user', 'system');
```

Comment:
Here again, the time zone must not be changed ([see above](https://github.com/concrete5/concrete5/pull/5017/files/78d4dad156b99c1638d1288942ad9c3330fbe828#r98235424))